app/tests/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+################################################################################
+# Tests
+################################################################################
+
+set(test_sources
+	main.cpp
+
+	test_operations.cpp
+)
+
+add_executable(app_unit_tests ${test_sources})
+
+target_link_libraries(app_unit_tests PUBLIC WildMeshingToolkit TetWild QuadricEdgeCollapse)
+target_link_libraries(app_unit_tests PUBLIC warnings::all)
+
+include(catch2)
+target_link_libraries(app_unit_tests PUBLIC Catch2::Catch2)
+
+
+# data()
+# set(DATA_DIR "${THIRD_PARTY_DIR}/data/")
+# target_compile_definitions(app_unit_tests PUBLIC -DWILDMESHING_TOOLKIT_DATA_DIR=\"${DATA_DIR}\")
+target_compile_definitions(app_unit_tests PUBLIC CATCH_CONFIG_ENABLE_BENCHMARKING)
+
+################################################################################
+# Sanitizers
+################################################################################
+
+if(WILDMESHING_TOOLKIT_WITH_SANITIZERS)
+	add_sanitizers(app_unit_tests)
+endif()
+
+################################################################################
+# Register tests
+################################################################################
+
+foreach(source IN ITEMS ${test_sources})
+    source_group("tests" FILES "${source}")
+endforeach()
+
+set(PARSE_CATCH_TESTS_ADD_TO_CONFIGURE_DEPENDS ON)
```

Comment:
This option is only for the alternative `ParseAndAddCatchTests()` function. This has no effect on `catch_discover_tests()`.

Reply:
done!